docs/public/configuration/backend.md

```diff
@@ -193,6 +202,43 @@ A US ASCII-only [PostgreSQL identifier]. Max length 63 characters.
 The name of the database to use.
 
 
+## CLICKHOUSE section (experimental)
+
+Available keys : `host`, `port`, `user`, `password`, `database`.
+
+### host
+
+An [LDH domain name] or IP address.
+
+The host name of the machine on which the PostgreSQL server is running.
+
+### port
+
+The [MySQL interface][Clickhouse MySQL interface]'s port the Clickhouse server
+is listening on.
```

Comment:
Why not just:

"The port the Clickhouse server is listening on."

I do not think the other information belongs here. Rather in the installation instruction.

Reply:
This is because Clickhouse can listen on several ports (several interfaces) at the same time. So this makes it explicit.

Reply:
OK, then it makes sense. Then that should maybe be mentioned in the installation document.